src/App.tsx

```diff
@@ -1,7 +1,11 @@
 import React from 'react';
-import './App.css';
+
 import {BrowserRouter, Route, Routes} from "react-router-dom";
 
+import './App.css';
+import ReportAnAdd from './pages/report-add';
```

Comment:
typo: should be ReportAnAd ? and report-ad, in the file names, URL, etc

Reply:
Hehe nice catch ! 
As long as I do not have computer at hand,
Either you can push this typo correction 
Or I merge the PR and you can provide a quick fix later ! 
Tell me :)

Reply:
done!